titan-web-client/.eslintrc.json

```diff
@@ -1,8 +1,8 @@
 {
-  "extends": ["semistandard", "semistandard-react"],
+  "extends": ["semistandard", "semistandard-react", "react-app"],
   "rules": {
     "jsx-quotes": ["error", "prefer-double"],
-    "react/prop-types": [1, { "ignore": ["children"] }]
+    "semistandard-react/prop-types": [1, { "ignore": ["children"] }]
```

Comment:
This was referencing the wrong lint rule, causing "children is missing in props validation" errors when running the app.